ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/ws/OpenIdConfiguration.java

```diff
@@ -71,4 +71,14 @@ public record OpenIdConfiguration(
     // OPTIONAL. JSON array containing a list of the JWE encryption algorithms (enc values)
     // supported by the OP for the ID Token to encode the Claims in a JWT [JWT].
     @JsonProperty("id_token_encryption_enc_values_supported")
-        List<String> idTokenEncryptionEncValuesSupported) {}
+        List<String> idTokenEncryptionEncValuesSupported,
+
+    // OPTIONAL. JSON array containing a list of Client Authentication methods supported by this
+    // Token Endpoint. The options are client_secret_post, client_secret_basic, client_secret_jwt,
+    // and private_key_jwt, as described in Section 9 of OpenID Connect Core 1.0 [OpenID.Core].
+    // Other authentication methods MAY be defined by extensions. If omitted, the default is
+    // client_secret_basic -- the HTTP Basic Authentication Scheme specified in Section 2.3.1 of
+    // OAuth 2.0 [RFC6749].
+    // See also: https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication
```

Comment:
aren't you supporting `private_key_jwt` only though? I'd use that as a default instead

Reply:
This is the generic model for an OpenIdConfiguration with the documentation from the spec. Yes, in reality for now it is always `private_key_jwt` but could easily be extended.
I also figured out the Keycloak actually just ignores it and need manual config :/